plugin-usage/adding-content/item-properties/events/README.md

```diff
@@ -65,3 +65,18 @@ You can make things happen ([actions](actions.md)) when an event is triggered (p
 * equipped\_armor\_item.player\_damaged\_by\_entity
 * bucket\_empty
 * bucket\_fill
+
+### EXAMPLE
+
+Why it's stated like `interact (dot) right`?
+Small example of how to state it into item config.
```

Comment:
```suggestion
Why is it written like `interact (dot) right`?
Below is a small example of how to correctly write it into your item config.
```


> Why is it written like `interact (dot) right`?

You never answer that question by the way, I would sort that out.

Reply:
The answer is that example.

Reply:
Interact.right
Is
Interact:
  Right:
Thats why the example is in there